t_frang/test_header_cnt.py

```diff
@@ -139,43 +140,66 @@ def test_settings_max_header_list_size(self):
         )
         self.assertEqual(client.h2_connection.remote_settings.max_header_list_size, 5)
 
-    def test_reaching_limit_headers_as_bytes(self):
+    def __test_reaching_limit_headers_as_bytes(self, huffman):
         """
         We set up for Tempesta `http_header_cnt 2` and
         made request with 4 headers as index from dynamic table.
         """
         client = self.base_scenario(
             frang_config="http_header_cnt 2;",
             requests=self.requests_with_same_header,
+            huffman=huffman,
         )
         client.responses.pop(0)
-        self.check_response(client, status_code="403", warning_msg=ERROR)
+        self.check_response(client, status_code="400", warning_msg=ERROR)
 
-    def test_not_reaching_limit_headers_as_bytes(self):
+    def test_reaching_limit_headers_as_bytes(self):
+        self.__test_reaching_limit_headers_as_bytes(True)
+
+    def test_reaching_limit_headers_as_bytes_no_huffman(self):
+        self.__test_reaching_limit_headers_as_bytes(False)
+
+    def __test_not_reaching_limit_headers_as_bytes(self, huffman):
         """
-        We set up for Tempesta `http_header_cnt 4` and
-        made request with 4 headers as index from dynamic table.
+        We set up for Tempesta `http_header_cnt 8` and
```

Comment:
Now wiki:

> http_header_cnt [NUM] Maximum number of HTTP headers in a HTTP message. 0 to disable the limit. Pseudo header from HTTP2 are ignored. Defaults: 0 (disabled).

Please change description.

Reply:
Yes Pseudo headers are not ignored